src/libostree/ostree-repo-pull.c

```diff
@@ -4658,7 +4658,9 @@ static void find_remotes_cb (GObject      *obj,
  * Pass the results to ostree_repo_pull_from_remotes_async() to pull the given @refs
  * from those remotes.
  *
- * No @options are currently supported.
+ * The following @options are currently defined:
+ *
+ *   * `override-commit-ids` (`as`): Array of specific commit IDs to fetch for refs
```

Comment:
You should document that this has to be the same length as `refs`, if provided, and that the indexing is the same.